packages/firestore/rollup.shared.js

```diff
@@ -145,6 +145,12 @@ const manglePrivatePropertiesOptions = {
     beautify: true
   },
   mangle: {
+    // Temporary hack fix for an issue where mangled code causes some downstream
+    // bundlers (Babel?) to confuse the same variable name in different scopes.
+    // This can be removed if the problem in the downstream library is fixed
+    // or if terser's mangler provides an option to avoid mangling everything
+    // that isn't a property.
```

Comment:
probably worth adding the issue link for future reference.

Reply:
Added.

Reply:
We have two lists of reserved properties now (see right below - line 158). Do we know that this will work?